boards/riscv32/hifive1/doc/hifive1.rst

```diff
@@ -29,38 +29,33 @@ to ``hifive1``.
 Flashing
 ========
 
-In order to upload the application to the device, you'll need OpenOCD and GDB
-with RISC-V support.
+In order to upload the application to the device, you'll need OpenOCD with
+RISC-V support.
 Download and installation instructions can be found in the
 `SiFive's Freedom-E-SDK GitHub repository
 <https://github.com/sifive/freedom-e-sdk>`_.
 
-With the necessary tools installed, you can connect to the board using OpenOCD.
-To establish an OpenOCD connection, switch to the
-:file:`utils` directory and run:
+Make sure your toolchain is using the correct version of OpenOCD.
```

Comment:
Can you write how to verify you have the "correct" version of OpenOCD?  Is there a version number you can check?

Reply:
My own recommendation would be to use the latest version of RISC-V OpenOCD distributed [on the SiFive website](https://www.sifive.com/boards).

The steps would be
1. Download the OpenOCD tarball for your OS
2. Extract the tarball
3. Pass `-DOPENOCD=path/to/riscv-openocd/bin/openocd` to cmake

We can also probably get rid of the reference to Freedom E SDK, since the only reason to pull that down is for the OpenOCD configuration, which would now be in Zephyr.

Reply:
I updated the pull request. Please let me know what you think.

Reply:
> Can you write how to verify you have the "correct" version of OpenOCD? Is there a version number you can check?

This is the version output from OpenOCD:
```
# zephyr sdk
$ openocd -v
Open On-Chip Debugger 0.10.0+dev-gea2753a7-dirty (2018-11-02-23:37)
Licensed under GNU GPL v2
For bug reports, read
	http://openocd.org/doc/doxygen/bugs.html
```
```
# risc-v
$ openocd -v
Open On-Chip Debugger 0.10.0+dev-gcd32bd4 (2018-12-19-01:15)
Licensed under GNU GPL v2
For bug reports, read
	http://openocd.org/doc/doxygen/bugs.html
```

Unfortunately that doesn't help. I guess you just have to know what you downloaded

Reply:
@dbkinder is there anything else you want me to change?